xla/stream_executor/rocm/hipblaslt_wrapper.h

```diff
@@ -23,6 +23,7 @@ limitations under the License.
 
 #if TF_HIPBLASLT
 #if TF_ROCM_VERSION >= 50500
+#include "rocm/include/hipblas/hipblas.h"
```

Comment:
Not sure if we need to support ROCM versions before 50500, but it might be on the safe side to move newly inclusion before "#if TF_ROCM_VERSION >= 50500" check (one line above).